core/src/main/java/dk/alexandra/fresco/framework/builder/numeric/PreprocessedValues.java

```diff
@@ -0,0 +1,23 @@
+package dk.alexandra.fresco.framework.builder.numeric;
+
+import dk.alexandra.fresco.framework.DRes;
+import dk.alexandra.fresco.framework.value.SInt;
+import java.util.List;
+
+/**
+ * Handles values needed internally within FRESCO which preferably should be preprocessed, but a
+ * default implementation could handle it online. In other words: protocol suites are encouraged to
+ * override the default implementation when implementing {@link BuilderFactoryNumeric}.
```

Comment:
"... override the default ... " - which you are neglecting in Spdz - see comment